tsconfig.json

```
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "target": "es6"
```

A few more options:

```suggestion
    "declaration": true,
    "module": "CommonJS",
    "inlineSourceMap": true,
    "strict": true,
    "target": "ES2015"
```

Using ES2015 is also a bit more clear than ES6, since the latter has a few different meanings in conversation.

---

Done in 02d2728